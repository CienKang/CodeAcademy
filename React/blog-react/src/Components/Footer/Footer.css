footer {
    background-color: black;
    color: white;
    width: 100vw;
    height: 6vh;
    display: flex;
    align-items: center;
}

footer p {
    margin-left: 15vw;
    font-size: 1.5rem;
}
