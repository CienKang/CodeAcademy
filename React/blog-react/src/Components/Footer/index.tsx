import React from 'react';

import './Footer.css';
const Footer = () :JSX.Element => {
    return (
        <footer>
            <p className="outer-margin-left-15">Â©artifact.com 2019</p>
        </footer>
    );
};

export default Footer;