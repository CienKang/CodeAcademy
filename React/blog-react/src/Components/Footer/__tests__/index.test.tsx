import React from "react";
import { render } from '@testing-library/react';
import Footer from '..';

describe('Footer', () => {

    it('should render the footer correctly when rendered ', () => {
        const {getByText} = render(<Footer />);
        expect(getByText('@artifact 2019')).toBeInTheDocument;
    });

    it('should match with snapshots when rendered for first time', () => {
        const {asFragment} = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });

});
