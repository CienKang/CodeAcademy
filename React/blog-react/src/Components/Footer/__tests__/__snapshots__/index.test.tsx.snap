// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match with snapshots when rendered for first time 1`] = `
<DocumentFragment>
  <div
    class="flex flex-row justify-start items-center bg-black h-24"
  >
    <p
      class="ml-32 text-white"
    >
      @artifact 2019
    </p>
  </div>
</DocumentFragment>
`;
